app/controllers/home_controller.rb

```diff
@@ -1,4 +1,11 @@
 class HomeController < ApplicationController
+  PAGES = %w(orders accounting production delivery store)
   def index
   end
+
+  def nav
+    page = PAGES.select { |p| p == params[:page] }.first
+    raise ActionController::RoutingError.new('Page Not Found') if page.blank?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.